lib/main.dart

```diff
@@ -1,67 +1,18 @@
 import 'package:flutter/material.dart';
-import '../../../utils/ui_constants.dart';
-import 'custom_widgets.dart';
-import '../../../models/movie.dart';
+import '../../../pages/movies_page.dart';
 
 void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
-  BoxDecoration buildBackdropDecoration(
-    Alignment begin,
-    Alignment end,
-  ) {
-    return BoxDecoration(
-      gradient: LinearGradient(
-        colors: UIConstants.backdropGradientColors,
-        begin: begin,
-        end: end,
-        stops: UIConstants.backdropGradientStops,
-      ),
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
-    Movie movie = Movie.fromStatic();
     return MaterialApp(
-      home: Scaffold(
-        backgroundColor: UIConstants.detailsBackgroundColor,
-        body: SafeArea(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: <Widget>[
-              AspectRatio(
-                aspectRatio: UIConstants.backdropAspectRatio,
-                child: DetailsBackdropSection(
-                  pathToBackdropImg: movie.pathToBackdropImg,
-                  genres: movie.genres,
-                  decoration: buildBackdropDecoration(
-                    Alignment.bottomCenter,
-                    Alignment.topCenter,
-                  ),
-                ),
-              ),
-              DetailsTitleSection(
-                movieTitle: movie.title,
-                movieReleaseDate: movie.releaseDate,
-                decoration: buildBackdropDecoration(
-                  Alignment.topCenter,
-                  Alignment.bottomCenter,
-                ),
-              ),
-              DetailsPosterSection(
-                pathToPosterImg: movie.pathToPosterImg,
-                movieScore: movie.score,
-              ),
-              MovieOverviewText(
-                movieOverviewTxt: movie.overview,
-              ),
-            ],
-          ),
-        ),
-      ),
+      initialRoute: MoviesPage.routeName,
+      routes: {
+        MoviesPage.routeName: (context) => const MoviesPage(),
+      },
```

Comment:
To debate: why do you define named routes and then navigate by pushing `MaterialPage`s? It'd be great to standardize a single approach to your app's navigation.

Reply:
I initially used named routes because it seemed like the de-facto more flexible choice. When it came to implementing the actual functionality though, I found that a stack modelled it perfectly, and that it was clearer to just say "push the Details page for this movie" without worrying about the route. Because of this I removed Details from the routes but left the Movies. At the moment, with only 2 pages, named routes don't really add anything. However later on if more pages (like a Favorites page) are added and we can reach Details from more than one place, it will be better for them to be named. If it's better to have a single routing scheme through the whole app, then it might be worth it to change Details to use a named route as well. Should I change this now or wait until there are actually more pages?